cmake/onnxruntime_providers.cmake

```diff
@@ -125,9 +125,13 @@ if (onnxruntime_USE_TENSORRT)
     set(PROTOBUF_LIBRARY libprotobuf)
     set(DISABLED_WARNINGS_FOR_TRT /wd4267 /wd4244 /wd4996)
   endif()
-  add_subdirectory(${ONNXRUNTIME_ROOT}/../cmake/external/onnx-tensorrt)
+  if ( CMAKE_COMPILER_IS_GNUCC )
+    set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-unused-parameter -Wno-missing-field-initializers")
```

Comment:
Are we disabling these warnings because of external trt code?

Reply:
Yes. Those warnings are only disabled for onnx-tensorrt parser.